ias_client/src/lib.rs

```diff
@@ -216,6 +217,19 @@ fn validate_quote_signature(
         return Err(QuoteVerificationError::InvalidCertificates(pem_certificates.to_string()));
     }
 
+    // TODO(KBS-174): remove this logging or make it a return, not a log
+    for (i, cert_bytes) in certificates.iter().enumerate() {
+        let res = X509::from_der(cert_bytes);
+        match res {
+            Ok(_) => {
+                // do nothing
+            }
+            Err(err) => eprintln!(
```

Comment:
stderr should be OK for a short term logging solution.  I checked both the fronend and replica systemd service -- they send stderr to syslog.  If this were to be permanent, I would suggest using the `log` crate and turning this into a `warn!()` macro call.